SPECS/ntopng/configure.ac.in.patch

```diff
@@ -0,0 +1,24 @@
+--- a/configure.ac.in     2023-11-02 14:11:55.000000000 +0000
++++ b/configure.ac.in     2024-02-13 21:07:24.667497169 +0000
+@@ -507,17 +507,12 @@
+ HIREDIS_INC=
+ HIREDIS_LIB=
+ AS_VAR_IF(with_hiredis, yes, [
+-    pkg-config --exists hiredis
```

Comment:
Do we want to change from using pkg-config to just directly checking for the package?

Reply:
The hiredis package is not installed in the current process. I am just building the hiredis source-code and installing the header and library files at the right location for ntopng build to use.

Can you clarify what you mean by "directly checking for the package" ??

Reply:
@nisamson  Can you please reply to this comment?